src/main/java/games/strategy/engine/framework/GameDataManager.java

```diff
@@ -107,7 +108,7 @@ public GameData loadGame(final InputStream inputStream, final String savegamePat
             return null;
           }
           final boolean closeCurrentInstance = buttonPressed.equals(yesClose);
-          GameRunner.startGame(savegamePath, newClassPath, null);
```

Comment:
@ssoloff noted this is the only place we call `startGame`, even easier then to remove the unnecessary third arg.